backend/src/api/ApiHandler.py

```diff
@@ -0,0 +1,56 @@
+import array
+from typing import Tuple
+
+from backend.src.MapAPI import MapAPI
+
+from backend.src.api.nominatim import get_lat_long
+from backend.src.api.overpass import nearby_point
+from backend.src.api.osrm import determine_travel_time
+from backend.src.api.osrm import TravelOptions
+
+
+class ApiHandler(MapAPI):
+
+    def convert(loc):
+        if not isinstance(loc, Tuple):
+            # Assume it's an address and convert
+            loc = get_lat_long(loc)
+        return loc
+
+    def get_travel_time(self, loc_1, loc_2) -> float:
+        loc_1 = self.convert(loc_1)
+        if loc_1 is None:
+            # TODO: What's an appropriate error here?
+            return None
+
+        loc_2 = self.convert(loc_2)
+        if loc_2 is None:
+            # TODO: What's an appropriate error here?
+            return None
+
+        travel_time = determine_travel_time(loc_1, loc_2, TravelOptions.WALK)
+        if travel_time is None:
+            # TODO: What's an appropriate error here?
+            return None
+
+        return travel_time
+
+    def get_nearby_options(self, loc, radius: float, n: int) -> array:
```

Comment:
this is my fault because my java brain but -> list instead of -> array, so we can remove the array import and fix MapAPI as well while we're at it

Reply:
Done!